core/src/actions/totals.ts

```diff
@@ -79,15 +79,14 @@ export class TotalsAction extends AuthenticatedAction {
         where: { createdAt: { [Op.gte]: new Date(dates[0]) } },
         group: [groupStatement],
       })
-    )
-      // @ts-ignore We use a custom group-by clause which returns more than one row as a result
-      .map((row) => {
-        const dateKey = Object.keys(row).find((r) => r !== "count");
-        return {
-          date: dateString(row[dateKey]),
-          count: row.count,
-        };
-      });
+    ).map((row) => {
+      const dateKey = Object.keys(row).find((r) => r !== "count");
+      return {
+        // @ts-ignore
```

Comment:
How come this needs to be ignored?

Reply:
because we added a `GROUP BY` above.  The new Seqeulize type fixed one thing (now `.count()` calls can return more than one row), but they broke another by assuming that the only 'column' returned is `"count"` .  In our case, we are grouping by by the date of the createdAt column, so the response actually looks something like:

```
[
{2020-01-01: 50},
{2020-01-02: 51}
]
```